.azure-pipelines/azure-pipelines-image-template.yml

```diff
@@ -45,7 +45,9 @@ jobs:
           ENABLE_DOCKER_BASE_PULL=y make PLATFORM=$(PLATFORM_AZP) PLATFORM_ARCH=$(PLATFORM_ARCH) configure
         displayName: 'Make configure'
     postSteps:
-      - publish: $(System.DefaultWorkingDirectory)/target
+      - script: cp target -r $(Build.ArtifactStagingDirectory)/
```

Comment:
Keep same file tree with former build template.

Reply:
What is "former build template"? I don't understand the motiviation and why it is required.

Reply:
Now the published artifacts is short of a directory 'target'. And we don't want to publish source code.